src/app/components/organism/footer/footer.component.css

```
@@ -58,10 +61,28 @@ router-link::ng-deep .foot-navigation:hover {
   margin-block-end: 2rem;
 }
 
+/* Estilos para el ícono de café */
+::ng-deep .coffee-icon {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: ng-deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)